lib/commands.js

```diff
@@ -169,10 +169,21 @@ async function run({ config, stack, minutes }, [ scenarioId ]) {
   const alertNames = arrayFrom(scenario.alerts, (i) => `${i}`.padStart(4, '0'))
   const queues = []
   for (const deployment of deployments) {
-    const queue = runQueue(alertNames, 20, async (alertName) => {
-      return await createAlert(deployment.kbUrl, alertName, 'ignored-for-now')
+    queues.push(async () => {
+      const createdActionId = await createAction(deployment.kbUrl, `index action`, `write-index`)
+      await runQueue(alertNames, 20, async (alertName) => {
+        return await createAlert(deployment.kbUrl, alertName, 'ignored-for-now', [{
+          id: createdActionId,
+          group: 'threshold met',
+          params: {
+            documents: [{
+              fromAlertId: '{{alertId}}',
```

Comment:
good start!

I'd like to extend this later, to see if we can get some dates in here, which could require that we add mappings as well.  We now have the date the action was scheduled in the variables via PR https://github.com/elastic/kibana/pull/83195, and then probably the current date, and probably the difference between the two.  We can then troll through this for another chart - "action execution lag" or such, probably with some kind of histogram or like one of the other scatterplots.

Reply:
That's true, I'll work on that. I'm going to add `executionTimeField: '@timestamp',` to the connector so it will set the date the connector executed and with what you mention, I can fiddle through the date scheduled!